mappings/servers/ezpvp/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "ezpvp",
+    "name": "EzPvP Network",
+    "description": "Kit-PvP and Practice Network based in Europe",

```

Comment:
GitHub Actions is requesting a shorter description that is 40 characters or less. Mind doing that please?

Suggestions:
- KitPvP and Practice Network based in EU
- KitPvP and Practice Network
- Kit-PvP and Practice Network based in EU

FYI: KitPvP is technically the correct way to spell it instead of Kit-PvP, so I wanted to present that

Reply:
ok should be good

Reply:
That works! I'll let GitHub actions run again. Hopefully should succeed.